Changelog.md

```diff
@@ -1,6 +1,42 @@
 ## Gazebo Msgs 11.x
 
-### Gazebo Msgs 11.x.x (202X-XX-XX)
+### Gazebo Msgs 11.0.0 (2024-08-26)
+
+Changes since 13.4.0:
```

Comment:
```suggestion
Changes since 10.2.0:
```

Reply:
Maybe, I just based it on the last version in the Changelog. I didn't check if there is a 10.3.0.